Src/GoogleApis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -22,19 +22,24 @@
 namespace Google.Apis.Auth.OAuth2
 {
     /// <summary>
```

Comment:
IMPORTANT!!!!
This interface should be in the Google.Apis.Auth and not in the DotNet4 project.
I didn't notice that before. My bad...